forms-flow-web/src/components/Form/List.js

```diff
@@ -287,7 +287,8 @@ const List = React.memo((props) => {
           {createDesigns && (
             <>
               <div className="d-md-flex justify-content-between align-items-center pb-3 flex-wrap">
-  <div className="d-md-flex align-items-center flex-grow-1 me-2">
+              <div className="d-md-flex align-items-center p-0 search-box input-group input-group" style={{ width: "320px" }}>
```

Comment:
please remove inline style and add that in MF?

Reply:
Updated